rpm/pacemaker.spec.in

```diff
@@ -415,6 +416,20 @@ manager.
 The %{name}-cluster-libs package contains cluster-aware shared
 libraries needed for nodes that will form part of the cluster nodes.
 
+%package -n %{python_name}-%{name}
+License:       LGPLv2+
+Summary:       Python libraries for Pacemaker
+Requires:      %{python_path}
+Requires:      %{pkgname_pcmk_libs} = %{version}-%{release}
+BuildArch:     noarch
+
+%description -n %{python_name}-%{name}
+Pacemaker is an advanced, scalable High-Availability cluster resource
+manager.
+
+The python3-pacemaker package contains a Python library that can be used
```

Comment:
"This package" to avoid hardcoding name

Reply:
We actually use "The %{name}-lib package" style strings throughout the spec file already.